site/source/getting-started/initial-setup.rst

```diff
@@ -37,17 +37,9 @@ Follow the instructions below to set up your server for the first time.
 		:width: 50%
 		:alt: SSD Initialization
 
-#. Click the ``Download`` button to download a file containing the URLs (`.local` and `.onion`) and Root CA of your server. Keep this file for future reference.
```

Comment:
Why did you delete this? It has not changed from 0.3.4.x. Users should continue to download their permanent address info.

Reply:
Because I was verifying in kisok mode and forgot that this has a different view. Will add back in.

Reply:
Restored